.changeset/rich-spiders-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Remove old `ToolbarBreadcrumbs`, `ToolbarBreadcrumbsClassKey` and `ToolbarBreadcrumbsProps`. Use new ones instead.
```

Comment:
To the users of comet, nothing really is removed, they just see that `ToolbarBreadcrumbs` now looks differently and may have different props/class-keys. 

```suggestion
Redesign the `ToolbarBreadcrumbs` component

Due to internal changes, including the props and class keys, custom usages and styling may need to be adjusted. 
```